pyClient/testZethUtils.py

```diff
@@ -0,0 +1,61 @@
+import zethMock
+import zethUtils
+
+from nacl.public import PrivateKey, PublicKey
+from nacl.encoding import HexEncoder
+
+# Tests that we get the correct PrivateKey object from the hexadecimal encoding
```

Comment:
I don't understand this test. We check that the output of a function is the same as the output of the same function (since get_private_key_from_hex (key) calls PrivateKey( key , encoder=HexEncoder).

Reply:
I will remove it, it made sense in the old version of the function. But not it does not.